Week-2/exercise-2.1/index.css

```diff
@@ -0,0 +1,125 @@
+* {
+  box-sizing: border-box;
+}
+body {
+  margin: 0;
+}
+.bg-container {
+  background-image: linear-gradient(
+    to right,
+    rgb(208, 78, 213),
+    rgb(74, 189, 218)
+  );
+  /*background-image: url("https://images.unsplash.com/photo-1536152470836-b943b246224c?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxleHBsb3JlLWZlZWR8Nnx8fGVufDB8fHx8&w=1000&q=80");*/
+  color: white;
+  background-size: cover;
+}
+header > h1 {
+  font-size: 40px;
+}
+header {
+  text-align: center;
+  padding-top: 20px;
+  font-family: "Courier New", Courier, monospace;
+}
+section {
+  text-align: center;
+  padding-top: 20px;
+  font-family: "Courier New", Courier, monospace;
+}
+.description {
+  margin: 70px;
+  margin-top: 30px;
+}
+img {
+  height: 30vh;
+  width: auto;
+  margin: 20px;
+}
+article {
+  background-color: rgb(7, 69, 96);
+  margin: 20px;
+  padding: 5px;
+  padding-left: 20px;
+  border-radius: 20px;
+}
+article > h1 {
+  margin-top: 10px;
+  margin-bottom: 10px;
+  font-size: 25px;
+  font-family: "Courier New", Courier, monospace;
+}
+article > p {
+  font-family: system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
+    Oxygen, Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
+  font-size: 15px;
+}
+a {
+  color: rgb(6, 6, 111);
+}
+ul {
+  background-color: rgb(7, 69, 96);
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  height: 50px;
+}
+li {
+  list-style-type: none;
+}
+.form-container {
+  background-image: linear-gradient(to bottom, #7b0d69, #b3269b);
```

Comment:
Already there is a background image to the container , no need to give this background image again.